finat/quadrature.py

```diff
@@ -90,6 +90,9 @@ def point_set(self):
     def weight_expression(self):
         return gem.Indexed(gem.Literal(self.weights), self.point_set.indices)
 
+    @property
```

Comment:
*suggestion*: Use a `cached_property` here, since `str`ifying the weights is potentially expensive.